synapse/tests/test_cortex.py

```diff
@@ -64,40 +75,21 @@ def getBaseModels():
                 ('foo:bar', {'subof': 'str', 'doc': 'A foo bar!'}),
             ),
             'forms': (
-                ('foo:bar',
-                 {'ptype': 'foo:bar'},
-                 []
-                 ),
+                ('foo:bar', {}, (
```

Comment:
Adding the defval prop here invalidates running the migration in CoreTestDataModelModuleV1 - while test_cortex_module_datamodel_migration test works its kind of pointless.